dev-requirements.txt

```diff
@@ -2,27 +2,30 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file=dev-requirements.txt dev-requirements.in setup.py
```

Comment:
Looks like `setup.py` was omitted when compiling these dev requirements.   See the section on pinning dev dependencies.   This repo is a library.
https://wiki.uchicago.edu/display/CDIS/GDC+Dependency+Management#GDCDependencyManagement-PinningDevDependencies